sentry/src/main/java/io/sentry/protocol/Geo.java

```diff
@@ -0,0 +1,167 @@
+package io.sentry.protocol;
+
+import io.sentry.ILogger;
+import io.sentry.JsonDeserializer;
+import io.sentry.JsonObjectReader;
+import io.sentry.JsonObjectWriter;
+import io.sentry.JsonSerializable;
+import io.sentry.JsonUnknown;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class Geo implements JsonUnknown, JsonSerializable {
+
+  /** Human readable city name. * */
+  @Nullable private String city;
+
+  /** Two-letter country code (ISO 3166-1 alpha-2). * */
+  @Nullable private String countryCode;
```

Comment:
@romtsn should we enforce this format somehow? Our unified docs mention this as well: https://develop.sentry.dev/sdk/event-payloads/user/#geographical-location

Reply:
I think it's fine, we have a few places already where we are expected to send data in certain format, but do not enforce it anyhow (e.g. `release`), we can think about it later I guess